jazz_slack-event-handler/components/config.js

```diff
@@ -0,0 +1,27 @@
+/**
+	Jazz slack evevnt handler
```

Comment:
seems like we want to follow one way of writing this file, here it is - https://github.com/tmobile/jazz/blob/v1.4.6/jazz_scm-webhook/components/config.js